lib/cavendish/node_version_ensurer.rb

```diff
@@ -0,0 +1,34 @@
+class VersionError < StandardError
+end
+
+module Cavendish
+  class NodeVersionEnsurer
+    def ensure!
+      raise VersionError.new(install_message) if installed_node_version.nil?
+      raise VersionError.new(update_message) if Cavendish::NODE_VERSION > installed_node_version
```

Comment:
If the installed node version is newer than canvendish node version, could that be a problem?